tests/TestCase/Model/Filter/ValueTest.php

```diff
@@ -46,6 +46,25 @@ public function testSkipProcess()
     /**
      * @return void
      */
+    public function testProcessWithoutField()
+    {
+        $articles = TableRegistry::get('Articles');
+        $manager = new Manager($articles);
+        $filter = new Value('title', $manager, [
+            'field' => []
```

Comment:
After a second look I think I now remember, the like filter needed some kind of protection because of possibly invalid/non-string values, and this protection could cause an empty set of conditions, ie this was actually a different situation.

Then the value filter got it too, as it was copied from the like filter, then I copied the value filter, from which in turn I spread it anywhere, and here we are now :)

So I guess in order to not break with the current behavior, that is, invalid queries being generated, causing a failure, I should revert this for all but the like filter, and then maybe an exception could be added for the next major?